.github/commands.json

```diff
@@ -17,5 +17,11 @@
     "name": "type/duplicate",
     "action": "close",
     "comment": "Thanks for creating this issue! It looks like this has already been reported by another user. We’ve closed this in favor of the existing one. Please consider adding any details you think is missing to that issue.\n\nTo avoid having your issue closed in the future, please read our [CONTRIBUTING](https://github.com/grafana/grafana/blob/master/CONTRIBUTING.md) guidelines.\n\nHappy graphing!"
-  }
+  },
+  {
+    "type": "label",
+    "name": "no new info",
+    "action": "close",
+    "comment": "This issue has been closed because it needs more information and has not had recent activity. please read our [CONTRIBUTING](https://github.com/grafana/grafana/blob/master/CONTRIBUTING.md) guidelines.\n\nHappy graphing!"
```

Comment:
```suggestion
    "comment": "This issue has been closed because it needs more information and has not had recent activity. Please read our [CONTRIBUTING](https://github.com/grafana/grafana/blob/master/CONTRIBUTING.md) guidelines.\n\nHappy graphing!"
```